lib/dbservice/batches.ex

```diff
@@ -118,4 +118,21 @@ defmodule Dbservice.Batches do
     |> Batch.changeset_update_users(users)
     |> Repo.update()
   end
+
+  @doc """
+  Updates the sessions mapped to a batch.
+  """
+  def update_sessions(batch_id, session_ids) when is_list(session_ids) do
```

Comment:
The Swagger PR has API docs, no? I feel like it will be needed :D

Reply:
Yes, coming after a couple of PRs.